src/act.ts

```diff
@@ -77,6 +74,14 @@ function withGlobalActEnvironment(actImplementation: Act) {
     }
   };
 }
+
+type MockAct = (callback: () => void) => void;
+type Act = typeof reactTestRendererAct extends undefined ? MockAct : ReactAct;
+
+const actMock = (callback: () => void) => {
```

Comment:
@thymikee do we still need to mock `act` in case RTR does not expose it? Seems like it was added in 2019. https://github.com/facebook/react/commit/267ed981466ef323b1461b013b5db8db31e31772

Reply:
Yeah let's drop it

Reply:
Dropping support and increasing peer dep req to React 16.8 (earlierst version to expose `act` in Test Renderer). I think we can avoid calling this a breaking change, as we already specified minimal RN version of 0.59, which uses React 16.8.3 ([see here](https://docs.google.com/presentation/d/1Z8mNkRw5CJ93qSQqESrcXbgia1dA8V1olnpKT0lsygk/edit#slide=id.gdc26277501_0_39))

Reply:
Sounds fair to me, I'd still suggest being explicit about it in the release notes!